.gitignore

```diff
@@ -2,7 +2,9 @@
 
 # Test outputs
 results.xml
-**/output.zarr
+*.zarr
+*.json
```

Comment:
That seems dangerous, json being a common configuration extension

Reply:
The issue is the example for performance numbers writes json timings to disk, and includes the timestamp in the file. I'll update to exclude only in the driver/examples directory.